assignments/callbacks.js

```diff
@@ -17,30 +17,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     return cb(arr[0]);
   }
 
-  // Function invocation 
+  // Function invocation
   firstItem(items, function(first) {
     console.log(first)
   });
 
-*/
 
 
+//work starts here
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+return cb(arr.length);
 }
```

Comment:
on these you may want to pass in the entire array into the callback, then inside the callback return the .length property of the arr